src/figuresClasses.ts

```diff
@@ -1,11 +1,84 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
 
-export class Triangle implements Figure {}
+  getArea: () => number;
+}
```

Comment:
you should specify the type of color and shape

Write an interface `Figure` and 3 classes implementing it so that every figure
has:
- a `shape` (`triangle`, `circle` or `rectangle`);
- a `color` (`red`, `green` or `blue`);